src/language/mixins.md

```diff
@@ -26,10 +26,17 @@ class Maestro extends Person [!with Musical, Aggressive, Demented!] {
 }
 {% endprettify %}
 
-To _implement_ a mixin, create a class that extends Object and
-declares no constructors.
-Unless you want your mixin to be usable as a regular class,
-use the `mixin` keyword instead of `class`.
+To _implement_ a mixin, use the `mixin` declaration. Or, if you want your mixin
```

Comment:
"Implementing a mixin" sounds a bit confusing to me (as "implementing" to me usually suggests implementing an interface). Should we just say "define a mixin"?

I wonder if we should be a bit more prescriptive about "mixin class" and say something like "In the rare case where you need to define both a mixin and a class, you can use the `mixin class` decl..."

Reply:
> I wonder if we should be a bit more prescriptive about "mixin class" 

I'm going to move the section I added on the topic to the class modifiers page, so that'll cover it